php/libraries/NDB_Notifier_Abstract.class.inc

```diff
@@ -194,10 +194,10 @@ abstract class NDB_Notifier_Abstract
      *
      * @return string   string in the form `First Last <email@domain.com>`
      */
-    private function _getLoggedInUser()
+    private function _getLoggedInUser(): string
```

Comment:
hey @johnsaigle you declared return type to be string as taken from the doc @return but the function actually returns an array. this is causing bugs in modules. how should we fix it? @driusan

Reply:
The easiest way to fix it is to change the return type in both to be array.. but I don't really see why it's returning an array and not the User object itself. (In fact, at that point the function is just returning User::singleton, so I'm not sure why this function is needed at all...)

Reply:
good question. i dont know, i'm not familiar with this class

Reply:
the same return type bug is happening with the function directly below this one as well

Reply:
Should be fixed in #4323.